### ISSUE TITLE
NYUV2 weight test SUNRGBD

### ISSUE THREAD

**Issue by zhouqunbing** [TUI-NICR/ESANet#55]
I trained a model on nyuv2,and i want to use the model to directly test on SUNRGBD dataset.But the classes in nyuv2 are 40 ,and the sunrgbd are 37,so the size in weight is not match.What should i do to solve the problem?
thanks!

**Comment by zhouqunbing** [comment]
Thanks, your thought is the same as mine, i delete the last convolution,if i use the cliped weights to test the SUNRGBD directly, the mIoU is very low ,so i need to load the weight for finetuning to get a better results, right?

**Comment by zhouqunbing** [comment]
meanwhile,if i train cityscape dataset,the loss,CrossEntropyLoss2d,should change the ignore_index from -1 to 255,right?.because the 255 is void.
![image](https://user-images.githubusercontent.com/92725408/233819456-c7ac9fcb-2334-4aac-b5b3-1f5d32ac533d.png)
